kzyGit/Weconnect-React
# Ft implement defence feedback

#### What does this PR do?
Improves user experience and adds tests

#### Description of Task to be completed?
- Improve tests to 31%
- Show user-friendly 404 error on invalid URLs
- User token in user validation
- Add loading to submit buttons when a request is made
- Update display of businesses
- Add HoundCI
- Add back navigation on every component
- Use base url from environment variable

#### How should this be manually tested?
- Start the server or open the application: ```https://github.com/kzyGit/Weconnect-React/tree/develop```
- Navigate through the functionality to ensure unnecessary popups are removed, user is unable to use invalid url and a user friendly error is shown on invalid urls
- Check on the repository's README to see the test coverage given by coverals

### What are the relevant pivotal tracker stories?

[#159476891](https://www.pivotaltracker.com/story/show/159476891)
